masschallenge/django-accelerator
# AC-8373 :: Delete old `gender` field

**Changes introduced in [AC-8373](https://masschallenge.atlassian.net/browse/AC-8373)**
- removed old gender field for the base_core_profile and test contexts
- updated failing tests
- this PR is shipped with https://github.com/masschallenge/accelerate/pull/2938

**How to test**
- deployed to test4
- ensure that all views that had the old gender field are working
- this includes all reports
1 - mentor pool report
2 - mentorship report
3 - judging report 
4 - staff bucketlists
5 - allocation stats
This list is not extensive will add more as I find them.
